pkg/controller/volume/scheduling/scheduler_binder_test.go

```diff
@@ -2053,7 +2063,10 @@ func TestBindPodVolumes(t *testing.T) {
 	}
 
 	for name, scenario := range scenarios {
-		t.Run(name, func(t *testing.T) { run(t, scenario) })
+		t.Run(name, func(t *testing.T) {
+			t.Parallel()
```

Comment:
Do we need `t.Parallel()` here and at the top of the Test function? I see the other test cases don't have the call here, and most examples I see have the parallel call inside the `t.Run()`

Reply:
Since this test function takes a minute here, add `t.Parallel()` to `t.Run()` to make all cases in this block parallel.

Reply:
Found that `scenario` is not defined causing shared state.
``` golang
for name, scenario := range scenarios {
      scenario := scenario
      t.Run(name, func(t *testing.T) {
	      t.Parallel()
	      run(t, scenario)
      })
}
```

Reply:
Thanks, Updated.

Reply:
Great find! Thanks!